Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -864,6 +864,9 @@ private extension PathHierarchy.Node {
     func fullNameOfValue(context: DocumentationContext) -> String {
         guard let identifier = identifier else { return name }
         if let symbol = symbol {
+            if let fragments = symbol[mixin: SymbolGraph.Symbol.DeclarationFragments.self]?.declarationFragments {
+                return fragments.map { $0.spelling }.joined().split(whereSeparator: { $0.isWhitespace || $0.isNewline }).joined(separator: " ")
+            }
             return context.symbolIndex[symbol.identifier.precise]!.name.description
```

Comment:
A small suggestion. But maybe we should prefer to use
```swift
if xx { 
    return A
} else {
    return B
}
``` 
than
```swift
if xx { 
    return A
}
return B
```

```suggestion
else {
    return context.symbolIndex[symbol.identifier.precise]!.name.description
}        
```

Reply:
We seem to be doing a bit of both styles throughout the code base. I don't have a strong preference for either style. When both the A and B are one line expressions (like in this case) I find the structure of the code easily to skim based on the leading `return` keywords so close to each other.

Reply:
Got it.